src/main/java/org/wso2/lsp4intellij/utils/DocumentUtils.java

```diff
@@ -38,6 +39,10 @@
     public static final String WIN_SEPARATOR = "\r\n";
     public static final String LINUX_SEPARATOR = "\n";
 
+    public static int getTabSize(Editor editor) {
```

Comment:
Lets move down this new util method, as its used only within the class (hence should be private).